doc/admin.md

```diff
@@ -68,6 +68,13 @@ occ config:app:set mail abuse_number_of_messages_per_1h --value=30
 occ config:app:set mail abuse_number_of_messages_per_1d --value=100
 ```
 
+## Google OAuth
+
+This app can allow users to connect their Google accounts with OAuth. This makes it possible to use accounts without 2FA or app password.
+
```

Comment:
Should we mention app passwords as well? Or are they not going to work any longer?

Reply:
They work, but only if you use 2FA.